.github/workflows/check.yaml

```diff
@@ -19,19 +23,17 @@ jobs:
         python-version: ["3.8", "3.10"]
     with:
       python-version: ${{ matrix.python-version }}
-      tox-version: "<4"
 
   func:
     uses: canonical/bootstack-actions/.github/workflows/func.yaml@v3
     needs: lint-unit
     strategy:
       fail-fast: false
     with:
-      command: "make functional"
+      command: "TEST_JUJU3=1 make functional"  # using TEST_JUJU3 due https://github.com/openstack-charmers/zaza/commit/af7eea953dd5d74d3d074fe67b5765dca3911ca6
       runs-on: "['self-hosted', 'xlarge', 'jammy', 'x64']"
```

Comment:
Can we use GitHub official runner, there is no need to use self-hosted. Thanks

Reply:
Fixed, thanks.